packages/form-js-editor/src/render/components/properties-panel/entries/ValuesSourceEntries.js

```diff
@@ -0,0 +1,219 @@
+import { get, isUndefined, without } from 'min-dash';
+
+import { SelectEntry, isSelectEntryEdited, TextFieldEntry } from '@bpmn-io/properties-panel';
+import ValueEntry from './ValueEntry';
+
+import { arrayAdd } from '../Util';
+import { useService } from '../../../hooks';
+
+export function ValuesSourceSelectorEntry(props) {
```

Comment:
Same as for the `groups`. This kind of breaks the current structures of one entry per file. Is there a specific reason why need to keep these entries together?

Reply:
Here I absolutely agree, these components should be separate and I'm currently doing just that